src/storage/standard_buffer_manager.cpp

```diff
@@ -522,6 +523,9 @@ void StandardBufferManager::DeleteTemporaryFile(block_id_t id) {
 	auto &fs = FileSystem::GetFileSystem(db);
 	auto path = GetTemporaryPath(id);
 	if (fs.FileExists(path)) {
+		auto handle = fs.OpenFile(path, FileFlags::FILE_FLAGS_READ);
```

Comment:
Can we close the handle after we have obtained the file size? Keeping it open might lead to locking conflicts upon removal.